packages/@react-spectrum/tabs/src/Tabs.tsx

```diff
@@ -279,7 +279,10 @@ export function TabList<T>(props: SpectrumTabListProps<T>) {
         {
           'spectrum-Tabs--quiet': isQuiet,
           ['spectrum-Tabs--compact']: density === 'compact'
-        })}>
+        },
+        styleProps.className
```

Comment:
noticed that `UNSAFE_className` wasn't being propagated through to TabList and TabPanels, fixed that here. Lemme know if this shouldn't be propagated